x/auth/keeper/grpc_query.go

```diff
@@ -3,6 +3,9 @@ package keeper
 import (
 	"context"
 
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	"github.com/cosmos/cosmos-sdk/types/query"
```

Comment:
could you sort the imports here?

Reply:
They already are? `make format` and `make lint` make them like so

Reply:
ah ok, weird, we generally have `stdlib imports || 3rd-party imports || cosmos-sdk imports`. It's okay then, probably that make command should be fixed.